tests/cockpit-tests

```diff
@@ -53,16 +48,41 @@ echo "Starting testing"
 iteration=0
 while true; do
     fail=0
-    find cockpit -name '*.py?' -delete
     git -C cockpit fetch origin
     git -C cockpit reset --hard origin/master
-    if [ "$iteration" -eq 0 ]; then
-        cockpit/test/vm-download --prune
-    else
-        cockpit/test/vm-download
-    fi
+
+    # TODO: Soon we'll move this to an images bot
     iteration=$(( (iteration+1) % 25 ))
-    if ! cockpit/test/github-task "$@"; then
-        sleep 60
+    if [ -x cockpit/test/vm-download ]; then
+        if [ "$iteration" -eq 0 ]; then
+            cockpit/test/vm-download --prune
+        fi
     fi
+
+    (
+        # If the tasks file is executable, run it
+        if [ -x cockpit/.tasks ]; then
+            cockpit/.tasks
+
+        # If it exists then cat it
+        elif [ -e cockpit/.tasks ]; then
+            cat cockpit/.tasks
+
+        # No tasks found
+        else
+            echo "echo 'No tasks found in cockpit repo' >&2; exit 1"
```

Comment:
OK, now I'm confused; if you want the `exit 1` to have any actual meaning, you need to set pipefail in the outer shell. `false | while read f; do echo $f; done` succeeds, doesn't actually iterate over anything, and does exactly the same as `true | while ...`. So in this caes there would be no sleep (and we don't want any either), but it would be the same (but cleaner) result with `exit 0`.

So, in summary, I think this is just a stylistic issue (and a bit confusing), I just want to make sure that you don't expect this `exit 1` to actually have a visible effect.